201-alert-to-slack-with-logic-app/azuredeploy.json

```diff
@@ -40,13 +78,20 @@
         },
         {
             "type": "Microsoft.Logic/workflows",
-            "apiVersion": "2016-06-01",
+            "apiVersion": "2015-02-01-preview",
```

Comment:
Why did we go backwards in time with the API? Is there no non-preview API we can use?